### ISSUE TITLE
Ticket type might contain whitespaces and isn't slugified

### ISSUE THREAD

**Issue by maltoe** [bitcrowd/tickety-tick#164]
Hey tickety-tick devs,

I just found out that I can't use the generated branch name as is (using Jira) , as we have a whitespace in the ticket type (`"tech story"`).

Would you think it would be ok to `slugify` the type by default as well?

https://github.com/bitcrowd/tickety-tick/blob/3175a51ca95daf1a4e2ff07ede04232efbbcde94/src/common/format/index.js#L8

Alternatively, adapt the Jira adapter to sanitize the type, besides lowercasing?

Thanks,
malte

❤️

**Comment by maltoe** [comment]
Ah, btw, the quotes don't help in this case, due to git

```sh
$ git checkout -b 'tech story/abc-1234-foo' && git commit --allow-empty -m '[#ABC-1234] Foo'
fatal: 'tech story/abc-1234-foo' is not a valid branch name.
```

**Comment by pmeinhardt** [comment]
Ah, thanks @maltoe 💚  We've not seen ticket types with whitespace before.
I think `slugify`ing sounds like a reasonable approach 👍 

Would be great if you can whip up a PR 🙂

**Comment by pmeinhardt** [comment]
Fixed via #166. Thanks a lot @maltoe 💚
